example-mlem-get-started/generate.sh

```diff
@@ -0,0 +1,156 @@
+#!/bin/bash
+# See https://dvc.org/doc/tutorials/get-started
+
+# Setup script env:
+#   e   Exit immediately if a command exits with a non-zero exit status.
+#   u   Treat unset variables as an error when substituting.
+#   x   Print commands and their arguments as they are executed.
+set -eux
+
+HERE="$( cd "$(dirname "$0")" ; pwd -P )"
+REPO_NAME="example-mlem-get-started"
+
+BUILD_PATH="$HERE/build"
+
+pushd $BUILD_PATH
+if [ ! -d "$BUILD_PATH/.venv" ]; then
+  exit 1
+  virtualenv -p python3 .venv
+  export VIRTUAL_ENV_DISABLE_PROMPT=true
+  source .venv/bin/activate
+  echo '.venv/' > .gitignore
+  pip install gitpython
+  pip install "git+https://github.com/iterative/mlem#egg=mlem[all]" --use-deprecated=legacy-resolver
```

Comment:
`--use-deprecated=legacy-resolver` why is it needed - out of curiosity? is it some known bug /problem?

Reply:
Yes, it's a known issue: https://github.com/iterative/mlem/issues/140

Reply:
It's faster and more importantly works (new resolver breaks until we resolve https://github.com/iterative/mlem/issues/140)

Reply:
yep, but does it mean that we will recommend the same in the Install guide and README? looks a bit weird ... (not critical, but I would try to address this before the release if possible)

Reply:
we'll deal with it before release

Reply:
also, should MLEM be installed from git here? Not from PyPi?

Reply:
we want git since we may want to prepare example repos before new release

Reply:
is it fixed?